docs/track.rst

```diff
@@ -215,6 +222,65 @@ Example::
         }
     ]
 
+data-streams
+............
+
+The ``data-streams`` section contains a list of all data streams that are used by this track.   Cannot be used if the ``indices`` section is specified.
+
+Each data stream in this list consists of the following properties:
+
+* ``name`` (mandatory): The name of the data-stream.
+
+Example::
+
+    "data-streams": [
+        {
+          "name": "my-logging-data-stream"
+        }
+    ]
+
+composable-templates
+....................
+
+The ``composable-templates`` section contains a list of all composable templates that Rally should create. These composable templates will often reference component templates which should also be declared first using the ``component-templates`` section.
+
+Each composable template in this list consists of the following properties:
+
+* ``name`` (mandatory): Composable template name
+* ``index-pattern`` (mandatory): Index pattern that matches the composable template. This must match the definition in the template file.
+* ``delete-matching-indices`` (optional, defaults to ``true``): Delete all indices that match the provided index pattern if the template is deleted.
+* ``template`` (mandatory): Composable template file name
+
+Example::
+
+    "composable-templates": [
+        {
+            "name": "my-default-composable-template",
+            "index-pattern": "my-index-*",
+            "delete-matching-indices": true,
+            "template": "composable-template.json"
+        }
+    ]
+
+component-templates
+....................
+
+The ``component-templates`` section contains a list of all component templates that Rally should create. These component templates will often be referenced by composable templates which can be declared using the ``composable-templates`` section.
+
+Each component template in this list consists of the following properties:
+
+* ``name`` (mandatory): Component template name
+* ``template`` (mandatory): Component template file name
+
+Example::
+
+    "component-templates": [
+        {
+            "name": "my-default-component-template",
+            "index-pattern": "my-index-*"
```

Comment:
The `index-pattern` property is not defined above. Did you mean to specify the `template` property here?

Reply:
good catch, thanks